README.md

```diff
@@ -843,6 +843,8 @@ Everything you need to become a Privacy Wizard.
 - [Privacy Tools](https://www.privacytools.io/) (thanks u/yieldingTemporarily)
 - [ethical.net](https://ethical.net/) (thanks u/yieldingTemporarily)
 - [AlternativeTo](https://alternativeto.net/) - Find alternatives for software. Used extensively to build this guide. Also check out [Switching.software](https://switching.software/) (thanks @mynamesleon)
+- [NoMoreGoogle.com](https://nomoregoogle.com/) - Another list of crowd sourced alternatives to google services. 
+- [Prism-Break.org](https://prism-break.org/) - Privacy conscious apps and services for every OS.   
```

Comment:
This is already in [Resources](https://github.com/tycrek/degoogle/blob/master/README.md#resources)

Reply:
sorry, just saw prism break. How about nomoregoogle.com?

Reply:
I've removed the duplicate link in the next  commit.

Reply:
> 
> 
> sorry, just saw prism break. How about nomoregoogle.com?

It appears to just be similar to this guide but less strict on what can be added (many of their listings are not private). Because of this I'm not too keen on adding it but I'll leave this open for discussion for now.

Additionally, and per my [previous comment](https://github.com/tycrek/degoogle/pull/223#pullrequestreview-473138073), don't directly edit the README. For this PR, edit https://github.com/tycrek/degoogle/blob/master/md/_resources.md